app/forms/placement_form.rb

```diff
@@ -44,16 +59,19 @@ def attributes=(attrs)
     end
   end
 
+  def school_name
+    school&.name if persisted?
```

Comment:
I think you can drop the `if`

Reply:
Done https://github.com/DFE-Digital/register-trainee-teachers/pull/3749/commits/69ef28a654a484a814a61c631e0076dc4f979022